### DISCUSSION TITLE
Support dropdown selection autocomplete

### DISCUSSION THREAD

**Discussion by zbjdonald** [directus/directus#9833] [Archive]
I have many options in dropdown interface, is it possible to add autocomplete feature in dropdown selection?

**Comment by rijkvanzanten** [comment]
Is this in regards to a "regular" string dropdown, or a many to one field?

**Reply by zbjdonald** [reply]
@rijkvanzanten Hi，just “regular” string dropdown.

**Reply by zbjdonald** [reply]
@rijkvanzanten Will that be a feature in the future ?

**Reply by rijkvanzanten** [reply]
Potentially! It's not in-works right now, but if there's enough community interest, a PR, or a party sponsoring the development it certainly can be 👍🏻

**Comment by iksent** [comment]
I am voting mostly for the autocomplete for languages select!

<img src="https://user-images.githubusercontent.com/32956521/153238187-5fd62ed5-c5b1-4ce1-83ff-66130efa7df0.png" width="400px"/>

Oh my! This is a real pain to search for a needed locale every time...

**Reply by benhaynes** [reply]
Yaaaass! Would you be open to a PR? ❤️

**Reply by zbjdonald** [reply]
Any updates？T。T

**Reply by nordcart** [reply]
And sometimes a m2o / o2m autocomplete could be useful instead of the drawer 🤔

**Reply by zbjdonald** [reply]
Any updates？T。T

**Reply by azrikahar** [reply]
@zbjdonald This was added in `9.13.0` via #13867. 🙂 Do note that it will only appear when there are more than 20 items (similar to how other search input only appears in navigation when there are a lot of collections)

**Reply by zbjdonald** [reply]
Thanks so much! I'll try it now.

**Comment by rijkvanzanten** [comment]
Heya! Thanks for opening this feature request!

This feature request has received over **15 votes** from the community. This means we'll move this feature request to the Under Review state! 

The Core team will schedule a meeting to review this request as soon as possible. The discussion will then be approved or denied. You may or may not be invited to join this meeting with the core team.

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._

**Comment by connorwinston** [comment]
This was implemented in version 9.13
